include/pybind11/pybind11.h

```diff
@@ -794,11 +794,16 @@ class module : public object {
     explicit module(const char *name, const char *doc = nullptr) {
         if (!options::show_user_defined_docstrings()) doc = nullptr;
 #if PY_MAJOR_VERSION >= 3
-        PyModuleDef *def = new PyModuleDef();
+        PyModuleDef *def = PyMem_New(PyModuleDef, 1);
         std::memset(def, 0, sizeof(PyModuleDef));
         def->m_name = name;
         def->m_doc = doc;
         def->m_size = -1;
+        def->m_free = [](void* module) {
+          if (module != nullptr) {
+            Py_XDECREF( PyModule_GetDef((PyObject*) module));
```

Comment:
extra space after '('

Reply:
Fixed. Note that neither of these were reported by `tools/check-style.sh`